File: JavaScript Fundamentals/Mid Exams/Mid Exam 2/numbers.js
function solve(string) {
    let numbers = string.split(' ').map(x => Number(x));
    let sum = 0;
    for (let num of numbers) {
        sum += num;
    }
    let average = sum / numbers.length;
    let topNumbers = [];
    for (let num of numbers) {
        if (num > average) topNumbers.push(num);
    }
    topNumbers.sort((a, b) => b-a);
    topNumbers = topNumbers.slice(0, 5);
    topNumbers.length > 0 ? console.log(topNumbers.join(' ')) : console.log('No');
}

File: Python Fundamentals/Mid Exam problems/numbers.py
sequence = list(map(int, input().split()))
average = sum(sequence) / len(sequence)
sequence.sort(reverse=True)

result_list = []

for element in sequence:
    if element > average:
        result_list.append(element)
        if len(result_list) == 5:
            break

if result_list:
    print(' '.join(list(map(str, result_list))))
else:
    print('No')
